int8 drone_id
---
bool proxy_success
